packages/runtime/src/index.ts

```diff
@@ -26,6 +26,10 @@ export interface RuntimeOptions {
    * Callback to modify config
    */
   configResolved?: (config: UserConfig, defaults: UserConfigDefaults) => void
+  /**
+   * Optional function to control UnoCSS style element(s) injection into DOM.
```

Comment:
```suggestion
   * Optional function to control UnoCSS style element(s) injection into DOM.
   * When provided, the default injection logic will be overridden.
```

Did I understand it right?

Reply:
yes! Well, maybe you can tell better if I missed something (see the comments about layer context etc. above)? But the initial DOM injection can be controlled like that and so far it's working great for me, I'm already using this in production and it does what I intended it to do. If there are more advanced use cases I didn't encounter yet, I'm glad to adapt to users needs.